docs/connectors/sqlserver.asciidoc

```diff
@@ -92,6 +80,122 @@ We welcome any feedback from a user to try the plug-in with database in managed
+  6. Resume the application
+  7. Start Debezium connector
+  8. When Debezium starts streaming from the new capture table it is possible to drop the old one using `sys.sp_cdc_disable_table` stored procedure with parameter `@capture_instance` set to the old capture instance name
+
+==== Hot schema update
+
+The hot schema update does not require any downtime in application and data processing.
+The procedure itself is also much simpler than in case of cold schema update
+
+  1. Apply all changes to the source table schema
+  2. Create a new capture table for the update source table using `sys.sp_cdc_enable_table` procedure with a unique value for parameter `@capture_instance`
+  3. When Debezium starts streaming from the new capture table it is possible to drop the old one using `sys.sp_cdc_disable_table` stored procedure with parameter `@capture_instance` set to the old capture instance name
+
+The hot schema update has one drawback.
+There is a period of time between the database schema update and creating the new capture instance.
+All changes that will arrive during this period will be captured by the old instance with the old structure.
+If your application does not tolerate such a transition period we recommend to follow the cold schema update.
+
+==== Example
+Deploy SQL Server based https://github.com/debezium/debezium-examples/tree/master/tutorial#using-sql-server[Debezium tutorial] to demonstrate the hot schema update.
+A column `phone_number` will be added to the `customers` table.
+
+[source,shell]
+----
+# Start the database shell
+docker-compose -f docker-compose-sqlserver.yaml exec sqlserver bash -c '/opt/mssql-tools/bin/sqlcmd -U sa -P $SA_PASSWORD -d testDB'
+----
+
+[source,sql]
+----
+-- Modify the source table schema
+ALTER TABLE customers ADD phone_number VARCHAR(32);
+
+-- Create the new capture instance
+EXEC sys.sp_cdc_enable_table @source_schema = 'dbo', @source_name = 'customers', @role_name = NULL, @supports_net_changes = 0, @capture_instance = 'dbo_customers_v2';
+GO
+
+-- Insert new data
+INSERT INTO customers(first_name,last_name,email,phone_number) VALUES ('John','Doe','john.doe@example.com', '+1-555-123456');
+GO
+----
+
+Kafka Connect log will contain messages
+```
+connect_1    | 2019-01-17 10:11:14,924 INFO   ||  Multiple capture instances ChangeTable [captureInstance=dbo_customers, sourceTableId=testDB.dbo.customers, changeTableId=testDB.cdc.dbo_customers_CT, startLsn=00000024:00000d98:0036, changeTableObjectId=1525580473, stopLsn=00000025:00000ef8:0048] and ChangeTable [captureInstance=dbo_customers_v2, sourceTableId=testDB.dbo.customers, changeTableId=testDB.cdc.dbo_customers_v2_CT, startLsn=00000025:00000ef8:0048, changeTableObjectId=1749581271, stopLsn=NULL] present for the same table   [io.debezium.connector.sqlserver.SqlServerStreamingChangeEventSource]
```

Comment:
What is "ChangeTable"? A placeholder for something?

Reply:
@gunnarmorling It is the name of the object - see `toString()` method

Reply:
Can you change this (in the code), so it reads like this:

> Multiple capture instances present for the same table:
> Capture instance "dbo_customers" (sourceTableId=..., ..., ...)
> Capture instance "dbo_customers_v2" (sourceTableId=...)